src/main/java/org/mongen/core/controller/OrganizationController.java

```diff
@@ -0,0 +1,53 @@
+package org.mongen.core.controller;
+
+import io.swagger.annotations.ApiOperation;
+import org.mongen.core.models.Organization;
+import org.mongen.core.service.OrganizationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1")
+public class OrganizationController {
+	@Autowired
+	OrganizationService organizationServ;
+	
+	@ApiOperation(value = "Get All the Organizations")
+	@GetMapping("/organizations")
+	public ResponseEntity<List<Organization>> getOrganizations(){
+		List<Organization> bene = organizationServ.getOrganizations();
+		return ResponseEntity.status(HttpStatus.OK).body(bene);
+	}
+	
+	@ApiOperation(value = "Get one Organization")
+	@GetMapping("/organization/{id}")
+	public ResponseEntity<Organization> getOrganizationId(@PathVariable("id") Long id){
+		Organization bene = organizationServ.findOrganizationById(id);
+		return ResponseEntity.status(HttpStatus.OK).body(bene);
```

Comment:
so if I call this endpoint, I get the json object, right?
I would love the one with `/organization/{name}` , as we will be showing name in the url, so I can grap it and search for it. For Visible Children i would expect `http://localhost:8080/api/v1/visible-children` to give me an object:
```
{
 id
 address
 country
 name
 verified
 mission
 vision
 logo
 tagline
 children {}
}
```
if it makes sense

Reply:
also, I would need an org info in most of endpoints, cause the page will look custom depending on the org name

Reply:
True, that's great to consider, will take a look into it

Reply:
You know, I think an endpoint like that should be handled in the frontend, and then from there request the information to the backend by POST, sending the organization name properly. The reason is that the name of an organization could use special characters and that could be messy if it's sent by GET encoded in the URL.

Other option could be that we generate a URL encoded name for the foundation, and then we store it in a field in the organization table

Reply:
let's have a quick catch up to discuss it maybe?
Cause it will be the base where we start from on the frontend, so would be good to decide on how do we go about it